applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS
+	bool "DVFS module"
+	depends on SOC_NRF54H20_CPUAPP
+	depends on CAF_BLE_COMMON_EVENTS || (BT_MAX_CONN = 1)
+	default y
+	help
+	  This option enable DVFS module which switches frequency and voltage according
+	  to application needs.
+
+if DESKTOP_DVFS
+
+config DESKTOP_DVFS_RETRY_BUSY_TIMEOUT_MS
+	int "Retry timeout"
+	default 50
```

Comment:
Have you checked if we can make this timeout shorter?

Reply:
according to lowlevel team it takes about 1ms for dvfs operation to be complete. However we can encounter situation where also other domain (secdom) requests dvfs change - in that situation lets set timeout for 2 ms

Reply:
Looks ok, 50 ms was far too much